templates/search/list.html

```diff
@@ -0,0 +1,51 @@
+{% from 'blog/macros.html' import article_list with context %}
+{% extends 'layout.html' %}
+
+{% block title %}Search results{% endblock %}
+
+{% block extra_css %}
+<link rel=stylesheet href=//netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css>
+<link rel=stylesheet href={{ url_for('static', filename='css/search.css') }}>
+<link rel=stylesheet href={{ url_for('static', filename='css/_article_list.css') }}>
+{% endblock %}
+
+{% block main %}
+<header>
+  <form action="{{ url_for('search') }}" method=get>
+    <input id=search type=search name=q placeholder="search for neomads and articles">
+    <input type=submit value=search>
+  </form>
+</header>
+
+<section>
+  {% if users %}
+  <article>
+    <h2>{{users|length}} neomads found</h2>
+    <ul>
+      {% for user in users %}
+      <li class=profiles>
+        <a href='{{ url_for_user(user) }}'>
+          <figure>
+            <img class=avatar-img src={{ user.avatar }} width=100 height=100 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
+          </figure>
+          <h2 name=username>{{ user.username }}</h2>
+        </a>
+      </li>
+      {% endfor %}
+    </ul>
+  </article>
+  {% endif %}
+  
+  {% if articles %}
+  <article>
+    <h2>{{articles|length}} articles found</h2>
+    <ul>
+      <li class=articles>
+        {{ article_list(articles=articles) }}
```

Comment:
`article_list` will be replaced by `<ul><li>…</li><li>…</li><li>…</li>…</ul>`

and here we insert this whole list into a `<ul><li>REPLACED LIST</li></ul>`

Therefore, we'll result with:

```html
<ul><li>
  <ul><li>…</li><li>…</li><li>…</li>…</ul>
</li></ul>
```
View the one single `li` in the first level that will contain a list of items.

In short, we don't want the `<ul><li>` at line 42, 43, 45, 46